src/pages/index.js

```diff
@@ -49,21 +59,17 @@ const popupCardForm = document.forms.newCardForm;
 
 const btnEditProfile = document.querySelector('.profile__btn_type_edit');
 const btnNewCard = document.querySelector('.profile__btn_type_add');
+const btnEditProfileImage = document.querySelector('.profile-photo-edit__edit-btn');
 
 const containerSelector = document.querySelector('.cards');
 
 const popupProfileForm = document.forms.profileForm;
 const popupPicture = document.querySelector('.popup_picture');
+const popupConfirmDelete = document.querySelector('.popup_confirm-delete');
+const popupEditProfileImage = document.querySelector('.popup_edit-profile-image');
+const EditProfileImageForm = document.forms.profileEditImageForm;
```

Comment:
Needs correction: Most developers in the industry and our code formatting standards recommend to use camelCase to name variables and functions. Please follow this recommendation to increase code readability